src/Pokedex.jsx

```diff
@@ -1,59 +1,27 @@
 import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import {
   AppBar,
   Toolbar,
   Grid,
   Card,
   CardMedia,
   CardContent,
-  CircularProgress,
   Typography,
   TextField,
 } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
 import SearchIcon from '@material-ui/icons/Search';
-import { fade, makeStyles } from '@material-ui/core/styles';
-import { toFirstCharUppercase } from './constants';
+import { makeStyles } from '@material-ui/core/styles';
+import { toFirstCharUppercase } from './utils/constants';
 import axios from 'axios';
 
-const useStyles = makeStyles((theme) => ({
```

Comment:
Pulled out all of these styles into their own file. Generally, you want to keep your styles and logic separate.